### PR TITLE
fix tracks collection from PV in TRK DQM

### PR DESCRIPTION
**Pull request by mtosi**
bug fix

### CODE REVIEW

**Comment by mtosi** [comment]
@cmsbuild, please test

**Comment by mtosi** [comment]
I think this does not happen because of this PR
may I ask to resubmit the test ?

thanks

**Comment by jfernan2** [comment]
please test

**Comment by boudoul** [comment]
Hi @mia, what is the impact of the bug fix - Is this used for data certification ? Thank you

File: DQM/TrackingMonitorSource/python/TrackCollections2monitor_cff.py [hunk 1]
```diff
@@ -233,47 +233,47 @@
...
-   zetaVtx        = cms.double(0.1),
-#   rhoVtx         = cms.double(0.2), ## tags used by b-tagging folks
-   rhoVtx         = cms.double(999.), ## tags used by b-tagging folks
-   nSigmaDtVertex = cms.double(0),
-   # should _not_ be used for the TrackWithVertexRefSelector
-   copyExtras       = cms.untracked.bool(False), ## copies also extras and rechits on RECO
-   copyTrajectories = cms.untracked.bool(False), # don't set this to true on AOD!
-)
+# the track collection
+trackAssociated2pvSelector.src = cms.InputTag('generalTracks')
```
Comment (makortel) [inline DQM/TrackingMonitorSource/python/TrackCollections2monitor_cff.py]: Let me just note that passing the overrides as parameters to `clone()` would reduce typing, and avoiding explicit types on the right-hand-side for already existing parameters would enable typo checks (i.e. along
```python
trackAssocated2pvSelector = trackWithVertexSelector.clone(
    # the track collection
    src = 'generalTracks',
    ...
)
```
would be better style)

**Comment by VinInn** [comment]
@boudoul , never used in validation or certification as "buggy" since ever (buggy==not-filtering-on-dz).
We hope it could be used in future as it should better pin-point tracks coming from the PV and exclude PU

**Comment by mtosi** [comment]
could we proceed, please ?
it would be great to have this integrated in the release we will use at Tier0

thanks !

**Comment by andrius-k** [comment]
+1

**Comment by fabiocos** [comment]
@mtosi @VinInn so I understand that the observed differences in DQM plots are what you expect

**Comment by fabiocos** [comment]
+1

**Comment by mtosi** [comment]
image

**Comment by mtosi** [comment]
thanks

**Pull request merged**